lib/ASTGen/Sources/ASTGen/ASTGen.swift

```diff
@@ -150,106 +133,218 @@ extension ASTGenVisitor {
 // Misc visits.
 // TODO: Some of these are called within a single file/method; we may want to move them to the respective files.
 extension ASTGenVisitor {
-  public func visit(_ node: MemberBlockItemSyntax) -> ASTNode {
-    visit(Syntax(node.decl))
+  public func generate(_ node: MemberBlockItemSyntax) -> ASTNode {
+    generate(node.decl)
   }
 
-  public func visit(_ node: InitializerClauseSyntax) -> ASTNode {
-    visit(node.value)
+  public func generate(_ node: InitializerClauseSyntax) -> ASTNode {
+    generate(node.value)
   }
 
-  public func visit(_ node: ConditionElementSyntax) -> ASTNode {
-    visit(node.condition)
+  public func generate(_ node: ConditionElementSyntax) -> ASTNode {
+    generate(node.condition)
   }
 
-  public func visit(_ node: CodeBlockItemSyntax) -> ASTNode {
-    visit(node.item)
+  public func generate(_ node: CodeBlockItemSyntax) -> ASTNode {
+    generate(node.item)
   }
 
-  public func visit(_ node: ArrayElementSyntax) -> ASTNode {
-    visit(node.expression)
+  public func generate(_ node: ArrayElementSyntax) -> ASTNode {
+    generate(node.expression)
+  }
+    
+  func generate(_ node: DeclSyntax) -> ASTNode {
+      return generate(Syntax(node))
+  }
+  func generate(_ node: ExprSyntax) -> ASTNode {
+    return generate(Syntax(node))
+  }
+  func generate(_ node: PatternSyntax) -> ASTNode {
+    return generate(Syntax(node))
+  }
+  func generate(_ node: StmtSyntax) -> ASTNode {
+    return generate(Syntax(node))
+  }
+  func generate(_ node: TypeSyntax) -> ASTNode {
+    return generate(Syntax(node))
+  }
+  func generate(_ node: some SyntaxChildChoices) -> ASTNode {
+    return self.generate(Syntax(node))
```

Comment:
Could you separate these functions by an empty line to be consistent with the style of the rest of the file?